src/sql/src/session/vars/definitions.rs

```diff
@@ -2006,6 +2006,13 @@ feature_flags!(
         internal: true,
         enable_for_item_parsing: false,
     },
+    {
+        name: enable_load_generator_upsert,
+        desc: "Create a LOAD GENERATOR UPSERT",
+        default: true,
```

Comment:
Insta-stabilizing this seems a bit scary? Since the primary need right now is for internal testing, seems safer to default `false` and stabilize it after we have some experience with it?